### ISSUE TITLE
[gatsby-transformer-toml] createContentDigest vs createDigestContent

### ISSUE THREAD

**Issue by arnau** [gatsbyjs/gatsby#13112]
## Description

gatsby-transformer-toml 2.1.8 uses `createDigestContent` which is not a function.

Note that if I edit the plugin in place to use `createContentDigest` (which is mentioned here https://www.gatsbyjs.org/docs/node-apis/) the result is a successful compilation.

### Steps to reproduce

Upgrade from 2.1.7 to 2.1.8.

### Expected result

Expected to compile without errors.

### Actual result

It fails with:

```
error Plugin gatsby-transformer-toml returned an error


  TypeError: createDigestContent is not a function

  - gatsby-node.js:39 Object.<anonymous>
    [specification]/[gatsby-transformer-toml]/gatsby-node.js:39:27

  - Generator.next


error Command failed with exit code 1.
```

### Environment

  System:
    OS: macOS 10.14.3
    CPU: x64 Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 11.13.0 - /usr/local/bin/node
    Yarn: 1.15.2 - /usr/local/bin/yarn
    npm: 6.7.0 - /usr/local/bin/npm
  Browsers:
    Chrome: 73.0.3683.86
    Firefox: 66.0.1
    Safari: 12.0.3
  npmPackages:
    gatsby: ^2.3.11 => 2.3.11
    gatsby-plugin-emotion: ^4.0.6 => 4.0.6
    gatsby-plugin-netlify: ^2.0.13 => 2.0.13
    gatsby-plugin-nprogress: ^2.0.10 => 2.0.10
    gatsby-plugin-react-helmet: ^3.0.11 => 3.0.11
    gatsby-plugin-typography: ^2.2.10 => 2.2.10
    gatsby-remark-autolink-headers: ^2.0.16 => 2.0.16
    gatsby-remark-copy-linked-files: ^2.0.11 => 2.0.11
    gatsby-remark-prismjs: ^3.2.6 => 3.2.6
    gatsby-source-filesystem: ^2.0.28 => 2.0.28
    gatsby-transformer-json: ^2.1.11 => 2.1.11
    gatsby-transformer-remark: ^2.3.8 => 2.3.8
    gatsby-transformer-toml: 2.1.8 => 2.1.8
    gatsby-transformer-yaml: ^2.1.11 => 2.1.11
  npmGlobalPackages:
    gatsby-cli: 1.1.58

**Comment by arnau** [comment]
This is the commit that introduces the problem https://github.com/gatsbyjs/gatsby/commit/980fa88816df1cf81c4113ee225a6fe69a9fa026

Which mentions `createContentDigest` and not `createDigestContent`.

**Comment by LekoArts** [comment]
Yeah, typo there. Would you be willing to put up a PR to fix the problem? Would highly appreciate that!

**Comment by arnau** [comment]
@LekoArts sure, find it here: #13118

**Comment by pieh** [comment]
Published `gatsby-transformer-toml@2.1.9`
